Community-meeting.md

```diff
@@ -0,0 +1,80 @@
+# First Community meeting
+
+
+## Goals
+
+Introduce FuseML to wider community, educate about its purpose and architecture.
+
+## Basic script
+
+- Introduction
+
+Welcome, state the goals of the meeting and agenda (very briefly)
+
+- Presentation/Demo (5-10 minutes)
+
+We could include same Demo Stefan/Alessandro already offered in different meetings to get people on the same page.
+
+- Installer
+
+Quick demonstration of the installer:
+
+  - mention that it is a fork of carrier project (meanwhile renamed to Epinio)
+    - https://github.com/epinio/epinio
+  - explain that it needs Kubernetes cluster with specific setup
+  - list the components parts it is installing
+  - show the installation
+
+- Brief architecture overview
+
+  (Unless it was already part of initial presentation point)
+
+  - Client-server architecture
+  - How it fits into MLOps mindset
+  - dependency on other components (mentioned by installer)
+  - show current components (more details to follow)
+
+
+- Concepts: Codesets
+
+  - "sets of code", basically the Data Scientist's project
+  - currently implemented as git(ea) repositories
+  - show commands live:
+    - create a codeset (+ show gitea web UI too)
+    - list codesets
+    - delete codeset
+
+- Concepts: Worfkflows
+
+  - This is the main component, driving MLOps work routine
+  - explain DSL, describe example yaml file
+  - currently implemented as Tekton pipelines
+  - explain relation with
+    - codesets
+    - workflow runs
+    - applications
+  - show commands live
+    - create workflow
+    - list codesets and assign workflow to codesets
+    - show workflow runs (+ show Tekton dashboard)
+  
+- Concepts: Applications
```

Comment:
Can we change the order by moving applications before workflows?

As workflows is what ties everything together, it might make sense to present the pieces first and them how workflows is able use/produce them.